packages/commonwealth/client/scripts/views/pages/discussions/helpers.ts

```diff
@@ -1,14 +1,15 @@
+import { NotificationCategories } from 'common-common/src/types';
 import { notifySuccess } from 'controllers/app/notifications';
 import moment from 'moment';
 import { Dispatch, SetStateAction } from 'react';
+import app from 'state';
 import type NotificationSubscription from '../../../models/NotificationSubscription';
 import type Thread from '../../../models/Thread';
-import { NotificationCategories } from 'common-common/src/types';
-import app from 'state';
-import { PopoverMenuItem } from '../../components/component_kit/cw_popover/cw_popover_menu';
 import { ThreadFeaturedFilterTypes } from '../../../models/types';
+import { PopoverMenuItem } from '../../components/component_kit/cw_popover/cw_popover_menu';
 
 export const getLastUpdated = (thread: Thread) => {
+  if (!thread) return undefined;
```

Comment:
@Miaplacidus I ran into a bug here when thread was undefined. This and line 33 are fixes. Just wanted to run them by you.

The bug occurred after creating a new Cosmos onchain proposal.

Reply:
Thanks, @mhagel! This makes sense. `CWContent` page isn't exclusively used for threads.